sources/web/datalab/metadata.ts

```diff
@@ -29,12 +29,18 @@ interface Creds {
 }
 
 interface FakeMetadata {
+  content_dir: string;
```

Comment:
I don't think these should be added to the FakeMetadata interface, since they are attributes of the server rather than the metadata contents.

Instead, pass in the settings object to the `launchFakeServer` function, and then use the values in the settings object.

Reply:
Done.